### PR TITLE
Fixed the Arm Deploy Retract feature

### PR DESCRIPTION
**Pull request by Rene-Baquero**

### CODE REVIEW

File: src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java [hunk 1]
```diff
@@ -8,33 +8,40 @@
...
+  private CANEncoder turnEncoder;
 
   //TODO: Make final once tuned
-  private static double kP = 0.1;
-  private static double kI = 0.0001;
-  private static double kD = 1.0;
+  private static double kP = 0.5;
+  private static double kI = 0.0;
+  private static double kD = 0.0;
   private static double kF = 0.0;
```
Comment (Rene-Baquero) [inline src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java:23]: Is there a point to having the ID and F values if they are all set at 0?

File: src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java [hunk 2]
```diff
@@ -16,7 +16,6 @@
 
   private CANSparkMax turnMotor;
   private CANPIDController turnController;
-  private CANEncoder turnEncoder;
 
   //TODO: Make final once tuned
   private static double kP = 0.5;
```
Comment (kushagharahi) [inline src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java]:
```suggestion
  private final double kP = 0.5;
```

**Review by kushagharahi — DISMISSED** [review]

File: src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java [hunk 3]
```diff
@@ -8,33 +8,38 @@
...
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
 import com.revrobotics.ControlType;
 
 public class ArmPidController {
 
+  private CANSparkMax turnMotor;
   private CANPIDController turnController;
 
   //TODO: Make final once tuned
```
Comment (kushagharahi) [inline src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java]:
```suggestion
```

File: src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java [hunk 4]
```diff
@@ -8,33 +8,38 @@
...
-  private static double kP = 0.1;
-  private static double kI = 0.0001;
-  private static double kD = 1.0;
+  private static double kP = 0.5;
+  private static double kI = 0.0;
```
Comment (kushagharahi) [inline src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java]:
```suggestion
  private final double kI = 0.0;
```

File: src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java [hunk 5]
```diff
@@ -8,33 +8,38 @@
...
+  private static double kD = 0.0;
```
Comment (kushagharahi) [inline src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java]:
```suggestion
  private final double kD = 0.0;
```

File: src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java [hunk 6]
```diff
@@ -8,33 +8,38 @@
...
   private static double kF = 0.0;
```
Comment (kushagharahi) [inline src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java]:
```suggestion
  private final double kF = 0.0;
```

File: src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java [hunk 7]
```diff
@@ -8,33 +8,38 @@
...
-    turnController.setReference(1, ControlType.kPosition);
+    turnController.setReference(0, ControlType.kPosition);
   }
 
   public void retract() {
-    turnController.setReference(-1, ControlType.kPosition);
+    turnController.setReference(-50, ControlType.kPosition);
   }
 
   public static double getKP() {
```
Comment (kushagharahi) [inline src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/manipulator/ArmPidController.java]: Can we get rid of all the getters and setters in this class, they are not needed anymore and were only there for tuning purposes

**Comment by kushagharahi** [comment]
Great work 👏 

Resolve the small comments I left and go ahead and merge :)

**Review by kushagharahi — APPROVED** [review]

**Pull request merged**